R/DEoptim.R

```diff
@@ -1,8 +1,8 @@
 DEoptim.control <- function(VTR = -Inf, strategy = 2, bs = FALSE, NP = 50,
                             itermax = 200, CR = 0.5, F = 0.8, trace = TRUE,
                             initialpop = NULL, storepopfrom = itermax + 1,
-                            storepopfreq = 1, checkWinner = FALSE,
-                            avWinner = TRUE, p = 0.2) {

```

Comment:
You may have mentioned this once before but I may have missed it.  _Why_ are we removing these two?  What is the motivation?   Catching up with DEoptim or something else?